client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -18,6 +24,9 @@ interface LowerJawPanelProps {
   resetButtonEvent: () => void;
   helpButtonEvent: () => void;
   hideHelpButton: boolean;
+  showShareButton: boolean;
+  superBlock: string;
+  block: string;
```

Comment:
The type can be cleaned if you import the shareProps

here is the syntax

```ts
interface LowerJawPanelProps extends ShareProps
```

```suggestion
```

Reply:
I didn't tried this way, and this is very good suggestion